mods/cnc/rules/campaign-maprules.yaml

```diff
@@ -66,3 +66,8 @@ airstrike.proxy:
 TRUCK:
 	Buildable:
 		Prerequisites: ~disabled
+
+TRAN:
+	WithCargoHatchAnimation:
+	Aircraft:
+		InitialFacing: 0
```

Comment:
This doesn't actually restrict the landing facing, so the door animation is inconsistently applied. This can be easily reproduced with the player controlled chinook in GDI06.

Reply:
It works for scripted moves because of
https://github.com/OpenRA/OpenRA/blob/23b3c237b7071fd308c4664b0b6c5d719c0f3c74/OpenRA.Mods.Common/Scripting/Global/ReinforcementsGlobal.cs#L159

Reply:
Can't we use `TurnToLand` for this?

Reply:
Yes, we can (and it requires exactly this setting to be set as well), but I would argue against it for usability and multiplayer rule consistency.